File: ktm-ui/src/app/models/CommonReqRes.ts
export class CommonReqRes {
    status: boolean;
    message: string;
    qty: number;
    data: any;
    type: string;
  
    constructor(status: boolean, message: string, qty: number, data: any, type: string) {
      this.status = status;
      this.message = message;
      this.qty = qty;
      this.data = data;
      this.type = type;
    }
  }

File: src/main/java/com/trading/kalyani/KPN/model/CommonReqRes.java
package com.trading.kalyani.KPN.model;

import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class CommonReqRes {

    private boolean status;
    private String message;
    private int qty;
    private Object data;
    private String type;


}
